static.config.js

```diff
@@ -19,6 +22,58 @@ export default {
       }),
       path: `/blog`,
     },
+    {
+      children: devops.map((dev) => ({
+        getData: () => ({
+          dev,
+        }),
+        path: `/dev/${dev.id}`,
+        template: `src/containers/Dev`,
+      })),
+      getData: () => ({
+        devops,
+      }),
+      path: `/DevOps`,
```

Comment:
```suggestion
      path: `/dev-ops`,
```
Continuing comment from navbar about URL standards